topology/probes/netlink.go

```diff
@@ -116,7 +116,7 @@ func (u *NetNsNetLinkProbe) handleIntfIsChild(intf *graph.Node, link netlink.Lin
 	}
 
 	if link.Attrs().ParentIndex != 0 {
-		if _, err := intf.GetFieldString("Vlan"); err == nil {
+		if _, found := intf.GetField("Vlan"); found {
```

Comment:
Maybe that would be better to fix the type of the vlan field to int64 line 375, so that we can use the GetFieldInt64 function here. This will help for Gremlin request to have the proper type.